api/src/dev/server/Dockerfile

```diff
@@ -45,5 +45,9 @@ RUN curl https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 \
 RUN curl https://raw.githubusercontent.com/mrako/wait-for/d9699cb9fe8a4622f05c4ee32adf2fd93239d005/wait-for \
   > /usr/local/bin/wait-for && chmod +x /usr/local/bin/wait-for
 
+RUN mkdir /.gradle && chmod a+rwx -R /.gradle
+VOLUME /.gradle
```

Comment:
I believe this makes the `/.gradle` directory exposed as a volume that can be imported by other containers. This could use a "why" comment.

Reply:
Commenting -- good point!

Reply:
Actually, I completely understood line 50 without the comment. What I was wondering was why the /.gradle directory should be exposed as a volume. That's where I was suggesting a "why" comment. (I never suggest "what" comments—if I didn't understand the code I'd suggest a change to the code rather than a comment.)

Reply:
Fair enough :-) I hope even my "what"ish comment got at the "why," too. The goal is to ensure that the cache "can be read/written by any container".

(Let me know if you want me to tweak in a follow-up.)